data/en/1_0005_updated.json

```diff
@@ -0,0 +1,2168 @@
+{
+    "data_version": "0.0.1",
+    "navigation": {
+        "visible": true,
+        "sections": [{
+                "title": "Introduction",
+                "group_order": ["introduction-group"]
+            },
+            {
+                "title": "Pay Pattern",
+                "group_order": ["pay-pattern"]
+            },
+            {
+                "title": "Weekly Pay",
+                "group_order": ["weekly-pay"]
+            },
+            {
+                "title": "Fortnightly Pay",
+                "group_order": ["fortnightly-pay"]
+            },
+            {
+                "title": "Calendar Monthly Pay",
+                "group_order": ["calendar-monthly"]
+            },
+            {
+                "title": "Four Weekly Pay",
+                "group_order": ["four-weekly-pay"]
+            },
+            {
+                "title": "Five Weekly Pay",
+                "group_order": ["five-weekly-pay"]
+            },
+            {
+                "title": "General Comments",
+                "group_order": ["general-comments-group"]
+            },
+            {
+                "title": "Summary",
+                "group_order": ["summary-group"]
+            }
+        ]
+    },
+    "groups": [{
+            "blocks": [{
+                "type": "Introduction",
+                "id": "introduction",
+                "primary_content": [{
+                    "id": "get-started",
+                    "content": [{
+                        "list": [
+                            "On average it takes ten minutes to complete once you’ve collected the information.",
+                            "Data should relate to all sites in England, Scotland and Wales.",
+                            "You can provide informed estimates if actual figures aren’t available.",
+                            "We will treat your data securely and confidentially."
+                        ]
+                    }]
+                }],
+                "preview_content": {
+                    "id": "preview",
+                    "title": "Information you need",
+                    "content": [{
+                        "title": "Employee pay frequency",
+                        "description": "You will need to indicate how frequently employees are paid. For example weekly, fortnightly, calendar monthly, four weekly or five weekly."
+                    }, {
+                        "description": "<strong>You will need the following information for all of your selected pay patterns:</strong>"
+                    }],
+                    "questions": [{
+                        "question": "Number of paid employees",
```

Comment:
missing text `If the last week of the month is affected by holidays please use a morte representative week` at the top of this group

Reply:
There is also a type in the spec "please use a **morte** representative week"